image_width: 800
image_height: 600
camera_name: wa1_dry
camera_matrix:
  rows: 3
  cols: 3
  data: [391.651079, 0.000000, 395.733051, 0.000000, 391.097930, 200.201610, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.242208, 0.038546, -0.002176, -0.001312, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [264.883087, 0.000000, 397.068545, 0.000000, 0.000000, 284.593994, 155.930718, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
